<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::CallSplash</class>
 <widget class="QWidget" name="ns::CallSplash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>198</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>198</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>198</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Incoming call</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetOuter {
	border: 1px solid #4D5F6D;
	border-radius: 3px;
}

#widgetOuter #widgetInner {
	border: 1px solid #BDE0F4;
	border-radius: 3px;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #9BD0F0, stop: 0.2 #EFF8FF);
}

#widgetCaption #labelTitle {
	font-size: 13px;
	font-weight: bold;
	color: #004362;
}

#widgetCaption #labelChannel {
	color: #083F56;
}

#widgetCaption #pushButtonClose {
	border: 0px;
	background-image: url(:/images/ui/splash_close.png);
}

#widgetCaption #pushButtonClose:hover {
	background-image: url(:/images/ui/splash_close_hover.png);
}

#widgetCaption #pushButtonClose:pressed {
	background-image: url(:/images/ui/splash_close_pressed.png);
}

#widgetMain {
	border-bottom: 1px solid #FAFEFF;
}

#widgetMain #widgetPhoto {
	border: 1px solid #00384F;
}

#widgetMain #widgetPhoto #labelPhoto {
	background: #88b6ef;
}

#widgetMain #labelDisplayName {
	color: #014260;
	font-weight: bold;
}

#widgetMain #labelCallerInfo {
	color: #3B3B3D;
}

#widgetMain #frameLine {
	background: #A8D5F2;
}

#widgetBottom {
	background: #E1F1FE;
	border-top: 1px solid #CEEAFF;
}

#widgetBottom #pushButtonAnswer {
	color: #FFFFFF;
	border-image: url(:/images/ui/splash_answer.png) 0 3 0 3;
	border-top: 0px transparent;
	border-bottom: 0px transparent;
	border-right: 3px transparent;
	border-left: 3px transparent;
}

#widgetBottom #pushButtonAnswer:hover {
	border-image: url(:/images/ui/splash_answer_hover.png) 0 3 0 3;
}

#widgetBottom #pushButtonAnswer:pressed {
	border-image: url(:/images/ui/splash_answer_pressed.png) 0 3 0 3;
}

#widgetBottom #pushButtonReject {
	border-image: url(:/images/ui/splash_decline.png) 0 3 0 3;
	border-top: 0px transparent;
	border-bottom: 0px transparent;
	border-right: 3px transparent;
	border-left: 3px transparent;
}

#widgetBottom #pushButtonReject:hover {
	border-image: url(:/images/ui/splash_decline_hover.png) 0 3 0 3;
}

#widgetBottom #pushButtonReject:pressed {
	border-image: url(:/images/ui/splash_decline_pressed.png) 0 3 0 3;
}

#widgetCountryLangValue {
	background:#57B5E1;
	border-top-left-radius: 15px;
	border-bottom-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-right-radius: 15px;
}

#labelCountryLangValue {
	font-weight: bold;
	font-size: 15px;
}

#labelCountryLang {
	color: #014260;
	font-size: 13px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetOuter" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetInner" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="QWidget" name="widgetCaption" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelTitle">
              <property name="text">
               <string>Incoming</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelTitleLine">
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelChannel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>226</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widgetSystemPanel" native="true">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>16777215</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButtonClose">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widgetMain" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="leftMargin">
             <number>15</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="widgetPhoto" native="true">
                <property name="minimumSize">
                 <size>
                  <width>68</width>
                  <height>68</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>68</width>
                  <height>68</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>1</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="labelPhoto">
                   <property name="minimumSize">
                    <size>
                     <width>66</width>
                     <height>66</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>66</width>
                     <height>66</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="leftMargin">
                 <number>9</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelDisplayName">
                  <property name="text">
                   <string>Barack Tasm</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelCallerInfo">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>39</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Mobile: %1</string>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QFrame" name="frameLine">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>1</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>1</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="labelCountryLang">
                <property name="text">
                 <string>Language:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widgetLanguage" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="labelSource">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelLine">
                   <property name="text">
                    <string>-</string>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelTarget">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widgetBottom" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>116</width>
                <height>16</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonReject">
              <property name="minimumSize">
               <size>
                <width>77</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>77</width>
                <height>22</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Decline</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAnswer">
              <property name="minimumSize">
               <size>
                <width>77</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>77</width>
                <height>22</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Answer</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections/>
</ui>
